src/applet/profile/download.c

```diff
@@ -9,24 +9,26 @@
 #include <euicc/es10a.h>
 #include <euicc/es10b.h>
 #include <euicc/es9p.h>
+#include <euicc/es9p_ex.h>
 #include <euicc/tostr.h>
 
-static const char *opt_string = "s:m:i:c:a:h?";
+static const char *opt_string = "s:m:i:c:a:ph?";
 
 static int applet_main(int argc, char **argv)
 {
     int fret;
-
     int opt;
 
     char *smdp = NULL;
     char *matchingId = NULL;
     char *imei = NULL;
     char *confirmation_code = NULL;
     char *activation_code = NULL;
+    int is_preview_mode = -1;
```

Comment:
It's better to use bool since we use C99. Incluse `stdbool.h` and use `bool` type, or use ugly `_Bool` directly.